lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -21,11 +46,16 @@ def initialize(serializer, options = {})
         def serializable_hash(options = nil)
           options ||= {}
 
-          if serializer.respond_to?(:each)
-            serializable_hash_for_collection(options)
-          else
-            serializable_hash_for_single_resource(options)
-          end
+          hash =
+            if serializer.respond_to?(:each)
+              serializable_hash_for_collection(options)
+            else
+              serializable_hash_for_single_resource(options)
+            end
+
+          ApiObjects::JsonApi.include_member? && hash[:jsonapi] = ApiObjects::JsonApi.member
```

Comment:
So, the one thing I was trying to avoid is `hash[:jsonapi] =` give the Adapter#serializable_hash the responsibility for knowing the jsonapi object key.  I was hoping the JsonApiObject would maintain all knowledge of building that object